tests/unit_tests/test_io_handler.py

```diff
@@ -25,6 +25,20 @@ def test_get_output_directory(args_dict, io_handler):
         label="test-io-handler", dir_type="model", test=True
     ) == Path(f"{args_dict['output_path']}/output/test-output/test-io-handler/model")
 
+    io_handler.plain_output_path = True
+
+    assert io_handler.get_output_directory(label="test-io-handler") == Path(
```

Comment:
we probably also want to get a warning (or info) when we pass both `label` and `plain_output_path` to inform the user that the latter is preferred over the first. and then I would also test the warning with caplog.

Reply:
Not sure about this - we generally don't check for every function the consistency of all arguments. In this case, the `use_plain_output_path` is clearly overwriting everything else and I would prefer not to add a info / warning.

Reply:
yes, in that case let's keep it like it is